### DISCUSSION TITLE
Pairing Code Not in Logs

### DISCUSSION THREAD

**Discussion by Azariah98** [maxileith/homebridge-appletv-enhanced#204] [Q&A]
> Every Apple TV is exposed as a Set-Top Box and is its own bridge. Therefore, we need to add every Apple TV seperatly to Apple Home. In order to do that, open the Home app, go to add devices > more options, then type in the pairing code from the logs (Logs: Please add [Apple TV Wohnzimmer (2)] manually in Home app. Setup Code: xxxx-xxxx this is not the code that you have seen on the Apple TV display).

I do not have this line in my logs. I do have all the lines before this showing that pairing was successful, up to and including, "[W] AppleTV (40:CB:C0:E9:1E:A0): Paring was successful. Add it to your home in the Home app: com.apple.home://launch"

I also see log entries showing that homebridge is successfully reading what is happening on the AppleTV.

-------------------------

```logs
1/25/2024, 6:10:54 PMAppletv Enhanced BridgeAppleTVEnhancedWARNING[W] AppleTV (40:CB:C0:E9:1E:A0): Paring was successful. Add it to your home in the Home app: com.apple.home://launch
1/25/2024, 6:10:54 PMAppletv Enhanced BridgeAppleTVEnhanced[I] AppleTV (40:CB:C0:E9:1E:A0): verifying credentials ...
1/25/2024, 6:10:58 PMAppletv Enhanced BridgeAppleTVEnhanced[I] AppleTV (40:CB:C0:E9:1E:A0): Credentials are still valid. Continuing ...
1/25/2024, 6:10:58 PMAppletv Enhanced BridgeAppleTVEnhanced[I] AppleTV (40:CB:C0:E9:1E:A0): Exposing Apple TV as accessory of type Apple TV.
1/25/2024, 6:11:02 PMAppletv Enhanced BridgeAppleTVEnhanced[I] Apple TV AppleTV (40:CB:C0:E9:1E:A0): Finished initializing
1/25/2024, 6:11:02 PMAppletv Enhanced BridgeApple TV AppleTV ED4D is running on port 38663.
1/25/2024, 6:11:05 PMAppletv Enhanced BridgeAppleTVEnhanced[I] Apple TV AppleTV (40:CB:C0:E9:1E:A0): New Active State: on
1/25/2024, 6:11:05 PMAppletv Enhanced BridgeAppleTVEnhanced[I] Apple TV AppleTV (40:CB:C0:E9:1E:A0): New Media Type State: video
1/25/2024, 6:11:05 PMAppletv Enhanced BridgeAppleTVEnhanced[I] Apple TV AppleTV (40:CB:C0:E9:1E:A0): Current App: com.plexapp.plex
```

**Comment by maxileith** [comment]
Duplicate of #208
